libraries/AP_Frsky_Telem/AP_Frsky_Telem.cpp

```diff
@@ -531,7 +524,7 @@ bool AP_Frsky_Telem::get_next_msg_chunk(void)
         uint8_t character = 0;
         _msg_chunk.chunk = 0; // clear the 4 bytes of the chunk buffer
 
-        for (int i = 3; i > -1 && _msg_chunk.char_index < sizeof(_statustext.next.text); i--) {
+        for (int8_t i = 3; i > -1 && _msg_chunk.char_index < sizeof(_statustext.next.text); i--) {
```

Comment:
ArduPilot's tradition is unsigned loop counters
```suggestion
        for (uint8_t i = 3; i >= 0 && _msg_chunk.char_index < sizeof(_statustext.next.text); i--) {
```

Reply:
...and it's a good tradition!